src/ORM/Hierarchy/Hierarchy.php

```diff
@@ -24,7 +24,6 @@
  *
  * @property int $ParentID
  * @property DataObject|Hierarchy $owner
- * @method DataObject Parent()
```

Comment:
See https://github.com/silverstripe/silverstripe-framework/pull/11092#discussion_r1424737200

Reply:
Good spot. I've done a search around looks like this is the only instance of get_extra_config() affecting relation config so manually fixing here is correct

Reply:
Sweet. I've added a note about it in the follow-up issue so CI doesn't flag it later on.